src/modules/heatmap/heatmapSettings.vue

```diff
@@ -77,7 +77,7 @@
             <label :for="ids.RadiusM">
                 {{ $m('radius') }}:
                 <span>
-                    {{ (settings[mode('RadiusM')] / 1000).toLocaleString() }}km
+                    {{ (settings[mode('RadiusM')]).toLocaleString() }}km
```

Comment:
this only adjust what's shown, input is still in meters and will also be processed by heatmap in meters.